### DISCUSSION TITLE
Embedding seed number parameters into PNG

### DISCUSSION THREAD

**Discussion by K3nt3L** [lllyasviel/Fooocus#512] [Ideas]
An element is missing to work on the reproducibility of images: recording the random seed number into the png file. Is it possible ? Do I miss something ? 
Thanks for making Foocus. :)

**Comment by barepixels** [comment]
This has been asked many time before, lllyasviel DOES NOT WANT prompt/seed metadata embedded in generated images from his version of Fooocus!

This is why many people like myself choose to use https://github.com/MoonRide303/Fooocus-MRE for serious works

**Reply by K3nt3L** [reply]
No problem for . Illyasvield's choice. But I see that we could find the seed into the log. I'am OK with that. 
Just want a log of my jobs. But I discovered after my post this log exists in a text file. :)

**Reply by mati-fernandez** [reply]
How can I use the seed to generate a new pic with same person?

**Comment by mashb1t** [comment]
https://github.com/lllyasviel/Fooocus/pull/1940 is planned to be included in release 2.2.0, publishing today/tomorrow. The feature is turned off by default.
